ocs_ci/ocs/node.py

```diff
@@ -1464,3 +1456,25 @@ def wait_for_new_osd_node(old_osd_node_names, timeout=180):
     except TimeoutExpiredError:
         log.warning(f"New osd node didn't appear after {timeout} seconds")
         return None
+
+
+def add_disk_to_node(node_obj, disk_size=None):
+    """
+    Add a new disk to a node
+
+    Args:
+        node_obj (ocs_ci.ocs.resources.ocs.OCS): The node object
+        disk_size (int): The size of the new disk to attach. If not specified,
+            the disk size will be equal to the size of the previous disk.
+
+    """
+    from ocs_ci.ocs.platform_nodes import PlatformNodesFactory
```

Comment:
Why was this import moved to inside this function?

Reply:
I put it here and not at the top of the file because it causes a circular loop error.
You can also see the same import at this line https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/node.py#L357.